libraries/anaconda3-solaris-0.1.2/context/solaris-build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
```

Comment:
Can we change this to `set -eu`? I feel like any script that expects the `ANACONDA_ENV` variable should error if that is not set